frontend/public/components/utils/service-level.tsx

```diff
@@ -0,0 +1,270 @@
+      loadServiceLevel(clusterIDParam);
+    }
+    // only onload an clusterID change
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [clusterIDParam]);
+
+  return [level, daysRemaining, loading];
+};
+
+const TrialDaysLeft: React.FC<{
+  level: string;
+  trialDaysLeft: number | null;
+  label?: boolean;
+}> = ({ level, trialDaysLeft, label }) => {
+  const { t } = useTranslation();
+
+  if ((level !== 'Eval' && level !== 'None') || (level === 'Eval' && trialDaysLeft === null)) {
+    return null;
+  }
+
+  let variant: 'warning' | 'danger' = 'warning';
+  let color: 'orange' | 'red' = 'orange';
+  let alertText = t('public~{{count}} day remaining', { count: trialDaysLeft });
+
+  if (trialDaysLeft === 0) {
+    alertText = t('public~< 1 day remaining');
+  }
+
+  if (trialDaysLeft < 0 || level === 'None') {
+    color = 'red';
+    variant = 'danger';
+    alertText = t('public~Trial expired');
+  }
+  if (label) {
+    return (
+      <div>
+        <Label
+          color={color}
+          icon={
+            color === 'orange' ? <YellowExclamationTriangleIcon /> : <RedExclamationCircleIcon />
+          }
+        >
+          {alertText}
+        </Label>
+      </div>
+    );
+  }
+  return (
+    <Alert
+      className="co-alert-inline--no-bold-title"
+      variant={variant}
+      isInline
+      isPlain
+      title={alertText}
+    />
+  );
+};
+
+export const ServiceLevelNotification: React.FC<{
+  clusterID?: string;
+  toggleNotificationDrawer: () => void;
+}> = ({ clusterID, toggleNotificationDrawer }) => {
+  const { t } = useTranslation();
+  const [level, trialDaysLeft] = useGetServiceLevel(clusterID);
+
+  if (!level || (level !== 'Eval' && level !== 'None')) {
+    return null;
+  }
+  let notificationStart = '';
+  if (level === 'None' || (level === 'Eval' && trialDaysLeft < 0)) {
+    notificationStart = t('public~Your 60-day self-support trial expired.');
+  }
+
+  notificationStart =
+    trialDaysLeft === 0
+      ? t('public~Your 60-day self-support trial will end in < 1 day.')
+      : t('public~Your 60-day self-support trial will end in {{count}} day.', {
+          count: trialDaysLeft,
+        });
+
+  return (
+    <NotificationEntry
+      title={t('public~This cluster is not supported.')}
+      actionExternalLinkPath={getOCMLink(clusterID)}
+      actionText={t('public~Get support')}
+      toggleNotificationDrawer={toggleNotificationDrawer}
+      description={
+        <>
+          {`${notificationStart} ${t(
+            'public~For continued support, upgrade your cluster or transfer cluster ownership to an account with an active subscription.',
+          )}`}
+        </>
+      }
+      type={NotificationTypes.warning}
+    />
+  );
+};
+
+type ServiceLevelProps = {
+  showOpenShiftManagerLink?: boolean;
+  clusterID?: string;
+  inline?: boolean;
+};
+
+export const ServiceLevel: React.FC<ServiceLevelProps> = ({
+  showOpenShiftManagerLink = true,
```

Comment:
I put it in that way because in most cases you want it to show - so defaulting it to true _maybe_ helps show that.  I also find negative a negative to be confusing.  So if you want the link to show the link (default state), the code is `!hideOCMLink`.

With that said, I can see your point

Reply:
✔️